specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+- Select a geographical area that the community covers, selecting from a list of already existing areas, preferably from a hierarchy of existing areas (global-continent-country-region-municipality-city)
+- Select which one language the community will be using
+- Select which vertical constellation the community will be focused on (multiselect, optional). If no constellation is selected then the geographical area covered should be very small (i.e. country or below)
+- The creator of the community becomes the default Council member by default
+
+## Actor/Action Matrix
+<!-- Describe which action is done by which actors. Feel free to use a table format or provide your own graphics. A "swimlane process chart" often works well here. -->
+REPLACE THIS TEXT
+
+|         | Admin | Member | Non-member Human | Non-member User |
 | ------- | -------- | -------- | -------- | -------- |
-| Actor 1 |          |          |          |          |
-| Actor 2 |          |          |          |          |
-| Actor 3 |          |          |          |          |
-| Actor 4 |          |          |          |          |
+| Find and Follow Community |          |          | Yes         | Yes         |
+| Join a Community |          |          | Yes         |          |
+| Approve a new Community member | Yes         |          |          |          |
+| Split a Community | Yes         |          |          |          |
+| Merge two Communities | Yes         |          |          |          |
+| Interact with my Community | Yes         | Yes         |          |          |
+| Ban a user from a Community | Yes         |          |          |          |
+| Manage a Community | Yes         |          |          |          |
+| Create a new Community |          |          | Yes         |          |
+
+## Data Architecture
+<!-- Describe the data architecture and principles -->
+
+### Geographies
+To prevent unmanageable complexity in the geography dimension, we propose that there should always be a "trunk" that sets the direction into narrower and narrower geographies, from which the language-branches and constellation-branches can expand. The trunk concept is the family of communities that don't have a constellation selected and that has English as their language. Only communities with these characteristics are allowed to create new "child" geographies. Conversely, any Commmunity that either has a non-english language or has narrowed down their Constellation will not be allowed to create new geographies. Instead they will be confined to only select pre-defined geographies from the trunk. 
+
+We should create an initial set of "trunk" Communities
+- Global
+   - North America
+      - Canada
+      - USA
+   - Asia
+      - Indonesia
+      - Japan
+      - South Korea
+      - Russia
+      - Vietnam
+   - South America
+   - Africa
+   - Europe
+      - DACH
+      - France
+      - Portugal
+      - Spain
+      - Ukraine
```

Comment:
How about UK, Italy, Balkans, Central Europe?... 

Lot of missing places in Asia as well.

Reply:
Overall, I think we should support every country, this split is too opinioinated.

Reply:
Of course we should support all, it should evolve as it wishes based on where people want their communities. The initial list was a minimum we should enable from day one, based on where Near communities are typically already strong.